src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/java/validators/annotations/AnnotationValidator.java

```diff
@@ -93,7 +93,10 @@ public String validate(String value, AnnotationAttributeRule rule) {
 				return null;
 			}
 			return rule.validate(value);
```

Comment:
For the code below, the usage of the try block is different, so I used standard exception handling for those cases.

Reply:
This looks structurally similar to what you refactored into the utility method. I'm not sure why the pattern wouldn't also apply here.

Reply:
Hi @mrglavas , I made the changes based on your suggestions